constants/megaMenuLinks.ts

```diff
@@ -0,0 +1,337 @@
+type SegmentData = {
```

Comment:
This definition already exists in `/constants/menuLinks.ts`. You don't need to create a new one